packages/core/src/Stores/client-store.js

```diff
@@ -1258,6 +1259,23 @@ export default class ClientStore extends BaseStore {
         }
     }
 
+    @action.bound
+    fetchConnectedApps() {
+        return new Promise(async (resolve, reject) => {
+            const response = await WS.send({ oauth_apps: 1 });
+            if (!response.error) {
+                console;
+                resolve(response.oauth_apps);
```

Comment:
You can set the store values via `setConnectedApps` in here to avoid having to call it in other places.

Reply:
I just want a function has one job, this one just fetch the connected apps and pass it to the place that we called it then we can decide to store it or do anything else, so I make it split but, if you think that would be more readable and better, I can change it. :pray: